packages/core/src/App/Components/Layout/Header/platform-switcher.jsx

```diff
@@ -61,11 +61,10 @@ const PlatformSwitcher = ({
                 <Icon
                     className='platform-switcher__icon'
                     icon={getPlatformInformation(app_routing_history).icon}
-                    size={32}
+                    alt={getPlatformInformation(app_routing_history).header}
```

Comment:
dont see "alt" as a prop in `icon.tsx`. Doesn't seem like icon.tsx was changed.

Reply:
on it 🙌🏼

Reply:
Check the fix in this [PR](https://github.com/hirad-rewok/deriv-app/pull/16)
cc: @hirad-rewok @matin-deriv

Reply:
Made the fix in this [PR](https://github.com/hirad-rewok/deriv-app/pull/16)
cc: @hirad-rewok @matin-deriv